src/include/distributed/multi_logical_optimizer.h

```diff
@@ -177,5 +177,6 @@ extern void FindReferencedTableColumn(Expr *columnExpression, List *parentQueryL
 extern char * WorkerColumnName(AttrNumber resno);
 extern bool IsGroupBySubsetOfDistinct(List *groupClauses, List *distinctClauses);
 extern bool TargetListHasAggregates(List *targetEntryList);
+extern Const * MakeIntegerConst(int32 integerValue);
```

Comment:
we should probably move this to some utility file

Reply:
this shouldn't be exported now since we are not setting any offset.
This should still go into a utility function but when I revert it, it will be untouched and probably a concern for a separate PR, there are other utilities like this such as `MakeIntegerConstInt64`.